barretenberg/bbup/.rebuild_patterns

```diff
@@ -0,0 +1,3 @@
+^barretenberg/bbup/bbup
+^barretenberg/bbup/install
```

Comment:
sorry botched this a bit - i was in the middle of making the commits trivially clickable :)

Reply:
All good, I've removed the extra entry.